applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -34,39 +33,35 @@ KRATOS_CREATE_LOCAL_FLAG( NodalConcentratedElement, COMPUTE_NODAL_INERTIA,
 KRATOS_CREATE_LOCAL_FLAG( NodalConcentratedElement, COMPUTE_DAMPING_RATIO,            4 );
```

Comment:
as i was telling, you should have two different elements, one for rotation and another one for diplacement.

also they should work out of the box, as read from the mdpa.

Reply:
I prefer to have only one element, the spring element integrates both. Do it the other way is inconsistent

Reply:
> also they should work out of the box, as read from the mdpa.

It works out of the box, you don't need to worry about that

Reply:
I am sorry but i cannot agree on the design with one single element. 

if you have two separate ones each of them is much simpler. why do you need to comine them into 1?? i really cannot see any advantage. and as i recall @philbucher was with me on this (please Philipp correct me if i am wrong)

Reply:
The advantage is that you don't need to create a new element, just modifying the properties it will add that behavior. Look at the code, if only displacement stiffness is defined the system is 3x3, if only rotation 3x3 but if both 6x6, and only modifying the properties/datavaluecontainer you will change the bahaviour, without dealing with the element creation

Reply:
BTW, this is consistent with the spring element https://github.com/KratosMultiphysics/Kratos/blob/277c550618dbe38965402e17ef2985a381f8ccec/applications/StructuralMechanicsApplication/custom_elements/spring_damper_element_3D2N.cpp#L121